content-repo/gendocs_test.py

```diff
@@ -100,6 +101,41 @@ def test_findfiles():
     assert f'{SAMPLE_CONTENT}/Integrations/integration-F5_README.md' in res
 
 
+CONTENT_INFO_TEST_INTEGRATIONS = [
+    (
+        f'{SAMPLE_CONTENT}/Deprecated_Integrations/ExportIndicators/ExportIndicators.yml',
+        f'{SAMPLE_CONTENT}/Deprecated_Integrations/ExportIndicators/README.md',
+        ':::caution Deprecated\nUse the Generic Export Indicators Service integration instead.\n:::\n\n'
+    ),
+    (
+        f'{SAMPLE_CONTENT}/Integrations/AnsibleAlibabaCloud/AnsibleAlibabaCloud.yml',
+        f'{SAMPLE_CONTENT}/Integrations/AnsibleAlibabaCloud/README.md',
+        ':::info Supported versions\nSupported Cortex XSOAR versions: 6.0.0 and later.\n:::\n\n'
+    ),
+]
+
+
+@pytest.mark.parametrize("yml_path, readme_path, expected_content_string", CONTENT_INFO_TEST_INTEGRATIONS)
+def test_add_content_info(yml_path, readme_path, expected_content_string):
+    """
+    Given -
+        a yml and readme path of integrations.
+
+        Case1: deprecated integration.
+        Case2: integration that is supported from version 6.0.0
+    When -
+        trying to fetch the integration information.
+    Then -
+        Case1: the content info will contain only information that the integration is deprecated.
+        Case2: the content info will contain only information that the integration is supported from 6.0.0 versions.
+    """
+    from gendocs import add_content_info
+    with open(yml_path, 'r', encoding='utf-8') as f:
```

Comment:
lets see if we can get rid of actual files for testing and use temp file created in the unit tests

Reply:
I have used tmp files but with minimal yml configuration required for the test, otherwise it would just be a really long dict to dump into the file without any real reason to do so.